oauthenticator/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .oauth2 import *
 from .github import *
 from .bitbucket import *
+from .mediawiki import *
```

Comment:
Another option would be for it to fail gracefully if it can not load mwoauth dependency (by just printing an appropriate message when the class is attempted to be used, rather than at import time)